app/services/section_service.js

```diff
@@ -0,0 +1,21 @@
+const { extractValidFields, validateAgainstSchema } = require('../../lib/validator')
+
+const sectionInformationSchema = {
+    id: {required: true},
+    courseId: {required: true},
+    number: {required: true},
+    joinCode: {required: true}
+}
+const sectionInsertSchema = {
+    courseId: {required: true},
+    number: {required: true},
+    joinCode: {required: true}
+}
+
+exports.extractSectionFields = (body) => {
+    return extractValidFields(body, sectionInformationSchema)
+}
+
+exports.validateSectionCreationRequest = (body) => {
```

Comment:
also validateSectionCreationRequest unused (but I'm going to work on sections so I can remove it later if I don't need it)